src/app/core/entity-list/entity-list/entity-list.component.html

```diff
@@ -170,7 +171,7 @@ <h2>{{ listName }}</h2>
 
   <button
```

Comment:
let's give users two options as actions, to either export all or only filtered data: "Download all data (.csv)" and "Download current (.csv)" (open to better wording, if anyone has suggestions).

Reply:
@sleidig, I have added one more button to `Download Current (.csv)`